config/cable.yml

```diff
@@ -12,6 +12,6 @@ production:
   adapter: solid_cable
   connects_to:
     database:
-      writing: cable
+      writing: production
```

Comment:
Setting this to production to allow the deploy to complete and the application start. This should be changed once we determine the multiple database stratagy.

Reply:
I've got a PR up for this, and a question out to Ops.